### DISCUSSION TITLE
manual-dados-mg/0.1/2.%20Ciclo%20de%20publica%C3%A7%C3%A3o%20de%20dados/016_semana_controle/

### DISCUSSION THREAD

**Discussion by giscus** [transparencia-mg/handbook#67] [Announcements]
# manual-dados-mg/0.1/2.%20Ciclo%20de%20publica%C3%A7%C3%A3o%20de%20dados/016_semana_controle/

https://transparencia-mg.github.io/manual-dados-mg/0.1/2.%20Ciclo%20de%20publica%C3%A7%C3%A3o%20de%20dados/016_semana_controle/

**Comment by Andrelamor** [comment]
Infelizmente o editor de vídeo usado na Oficina de 10/05 não captou a transição de slides no link 'Vídeo', mas a apresentação pode ser baixada e revista
